eras/alonzo/test-suite/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-alonzo-test`
 
+## 1.3.0.1
```

Comment:
Should this be `1.3.0.0`?

Reply:
Mhm, good spot . So, the entry that  I added  (1.3.0.1) is correct - because what [we released is 1.3.0.0 ](https://github.com/IntersectMBO/cardano-haskell-packages/tree/main/_sources/cardano-ledger-alonzo-test/1.3.0.0). 
But yes, the changelog entry for 1.3.0.0 is missing entirely - so I will add it, will try to see if i can find what the changes are. Thanks for spotting it!

Reply:
Done, thanks!